examples/webgl_geometry_teapot_collada.html

```diff
@@ -0,0 +1,420 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - teapot buffer geometry</title>
```

Comment:
```html
<title>three.js webgl - collada exporter</title>
```
and probably rename file to `misc_exporter_collada.html`

Reply:
I made the 2 changes above and just noticed the files.js
Should I add this to the "misc" section of the list?

Reply:
Yes, otherwise the example won't be visible on the overview page: https://threejs.org/examples/